docs/rpi3.md

```diff
@@ -292,19 +292,20 @@ need to reboot the RPi after a rebuild.
 # 6. OpenOCD and JTAG
 First a word of warning here, even though this seems to be working quite good as
 of now, it should be well understood that this is based on incomplete and out of
-tree patches. So what are the major changes that enables this? First [OpenOCD]
-currently doesn't contain ARMv8-A / AArch64 support in the upstream tree. A
-couple of different people have put something together that gets the job done.
-But to get in a shape for upstream, there is still quite a lot left to do. The
-other change needed is in U-Boot, that is where we configure the [RPi3 GPIO
-pins] so that they will talk JTAG. The pin configuration and the wiring for the
-cable looks like this:
+tree patches. Indeed, [OpenOCD] currently doesn't contain ARMv8-A / AArch64
```

Comment:
> I'm not sure if a RPi3 SoC config file was merged though

it's not merged (there are config files only for rpi and rpi2), just checked it

@jbech-linaro my suggestion - I can try to check if debugging works with the mainline OpenOCD (as support of armv8 is merged), if it's OK - I'll change information about seqlabs OpenOCD fork and just add a notice that rpi3 config from our rep should be used  https://github.com/OP-TEE/build/blob/master/rpi3/debugger/pi3.cfg